### DISCUSSION TITLE
V3

### DISCUSSION THREAD

**Discussion by danielducu** [Schluggi/AIOsense#191] [Q&A]
Can you please update us on when the V3 is comming out? Thanks :)

**Comment by Schluggi** [comment]
Hi, sure. I just merged it. RC1 will be released in a couple of minutes.

**Reply by danielducu** [reply]
But when do we get to see the v3 board :) ? Case ? Thanks

**Reply by Schluggi** [reply]
The V3-RC1 boards just arrived. So I'll test and if there are no issues, I will release V3. Feel free to order V3-RC1 as well and give Feedback.

Images of the board will be available with the final V3 release (mid April, in case of no issues)

**Reply by s-gordon** [reply]
How are things looking so far with the testing?

**Reply by s-gordon** [reply]
@Schluggi 

I had a go at assembling this on a breadboard last weekend. The amp and speaker work wonderfully, but no luck with the mic and voice assist.

**Reply by AnthonyPluth** [reply]
Has anyone had any success with the I2S mic? I ordered a handful of v3 boards and sispeed mics but can't seem to get the mics working

**Reply by s-gordon** [reply]
I couldn't get the mics working either.

**Reply by AnthonyPluth** [reply]
Figured out my issue - I didn't have my voice assistant fully configured in Home Assistant...
